src/server/game/Entities/Player/Player.h

```diff
@@ -1242,6 +1242,7 @@ class Player : public Unit, public GridObject<Player>
     [[nodiscard]] Item* GetItemByPos(uint16 pos) const;
     [[nodiscard]] Item* GetItemByPos(uint8 bag, uint8 slot) const;
     [[nodiscard]] Bag*  GetBagByPos(uint8 slot) const;
+    uint32 GetFreeInventorySpace() const;
```

Comment:
shouldn't this be ` [[nodiscard]]` as well?

Reply:
I dont know, I didnt want to put it to wait feedback since I saw some didn't use [[nodiscard]]

Reply:
gonna add